sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -0,0 +1,171 @@
+      testingSplitPercentage: 30,
+      trainingSplitPercentage: 70,
+    },
+  },
+};
+const deployParam: TextAnalysisAuthoringDeployProjectParameters = {
+  body: {
+    trainedModelLabel: modelLabel,
+  },
+};
+
+async function createStorageBlob(): Promise<ContainerClient> {
+  /* Return a container client */
+  const defaultAzureCredential = new DefaultAzureCredential();
+  // Get documents containers
+  const blobServiceClient = new BlobServiceClient(
+    assertEnvironmentVariable("STORAGE_ENDPOINT"),
+    defaultAzureCredential
+  );
+  return blobServiceClient.getContainerClient(storageInputContainerName);
+}
+
+async function uploadDocumentsToStorage(dirName: string): Promise<void> {
+  const fileNames = await readdir(dirName);
+  const client = await createStorageBlob();
+  for (const file of fileNames) {
+    if (file.endsWith(".txt")) {
+      const blob = client.getBlockBlobClient(file);
+      const filePath = path.join(dirName, file);
+      await blob.uploadFile(filePath);
+    }
+  }
+  rm(dirName, { recursive: true, force: true });
+}
+
+async function getAssetsforProject(
+  projectKind:
+    | "CustomEntityRecognition"
+    | "CustomSingleLabelClassification"
+    | "CustomMultiLabelClassification"
+): Promise<{
+  assets:
+    | ExportedCustomSingleLabelClassificationProjectAssets
+    | ExportedCustomMultiLabelClassificationProjectAssets
+    | ExportedCustomEntityRecognitionProjectAssets;
+  dirName: string;
+}> {
+  switch (projectKind) {
+    case "CustomSingleLabelClassification":
+      return { assets: customSingleLabelAssets, dirName: path.join(pathName, "WebOfScience") };
+
+    case "CustomMultiLabelClassification":
+      return { assets: customMultiLabelAssets, dirName: path.join(pathName, "MoviesSummary") };
+
+    case "CustomEntityRecognition":
+      return { assets: customEntityAssets, dirName: path.join(pathName, "LoanAgreements") };
+  }
+}
+
+async function polling(client: TextAuthoringClient, project: any) {
+  const poller = getLongRunningPoller(client, project);
+  await poller.pollUntilDone();
+}
+
+export async function createCustomTestProject(
+  client: TextAuthoringClient,
+  projectKind:
+    | "CustomEntityRecognition"
+    | "CustomSingleLabelClassification"
+    | "CustomMultiLabelClassification",
+  projectName: string,
+  deploymentName: string
+): Promise<void> {
+  const { assets, dirName } = await getAssetsforProject(projectKind);
+  await decompress(dirName.concat(".zip"), pathName);
+  await uploadDocumentsToStorage(dirName);
+
+  const createProjectOptions: CreateProjectOptions = {
+    projectKind,
+    storageInputContainerName,
+    projectName,
+    language,
+    multilingual: false,
+    settings: {},
+  };
+
+  // Create a project
+  const importTask = await client
+    .path("/authoring/analyze-text/projects/{projectName}/:import", projectName)
+    .post({
+      body: {
+        metadata: createProjectOptions,
+        projectFileVersion: "2022-05-01",
```

Comment:
Could we pull this into a top-level constant so we can update it later for recent GA versions?

Reply:
We can use one constant for this and for `trainingConfigVersion`.